forge-gui/res/cardsfolder/upcoming/zur_eternal_schemer.txt

```diff
@@ -0,0 +1,11 @@
+Name:Zur, Eternal Schemer
+ManaCost:W U B
+Types:Legendary Creature Human Wizard
+PT:1/4
+K:Flying
+S:Mode$ Continuous | Affected$ Creature.Enchantment+YouCtrl | AddKeyword$ Deathtouch & Lifelink & Hexproof | Description$ Enchantment creatures you control have deathtouch, lifelink, and hexproof.
+A:AB$ Animate | Cost$ 1 W | ValidTgts$ Enchantment.nonAura+YouCtrl | TgtPrompt$ Select target non-Aura enchantment you control | Power$ X | Toughness$ X | Types$ Creature | Duration$ Permanent | StackDescription$ SpellDescription | SpellDescription$ Target non-Aura enchantment you control becomes a creature in addition to its other types and has base power and toughness each equal to its mana value.
```

Comment:
How about – `StackDescription$ {c:Targeted} becomes a creature in addition to its other types and has base power and toughness each equal to its mana value.`

I need to take a look at better parsing for Animate stuff, it's kind of a mess

Reply:
That should look neater. Made the change